docs/integrations/code-conventions.md

```diff
@@ -600,6 +600,34 @@ You may also use ```headerTransform``` to convert the existing keys into formatt
  
  <img width="758" src="../doc_imgs/integrations/50575199-fd5d0a00-0e01-11e9-9d54-944eb7c6f287.png"></img>
 
+### IndicatorsTimeline
+The `IndicatorTimeline` is an optional object (available from Server version 5.5.0 and up) . It is only applicable for commands that operate on indicators. It is a dictionary (or list of dictionaries) of the following format:
+When `IndicatorTimeline` data is returned in an entry, the timeline section of the indicator whose value was noted in the timeline data will be updated (and is viewable in the indicator's view page in Cortex XSOAR as can be seen in the attached image).
+
+<img width="758" src="../doc_imgs/integrations/timeline_section.png"></img>
+
+**What value should be used for the `'Category'` field of a `timeline` data object?**  
+Any Cortex XSOAR integration command that returns `timeline` data should include the `'Category'` value of `'Integration Update'`. When returning `timeline` data from a Cortex XSOAR automation, the value passed to the `'Category'` field should be `'Automation Update'`.
```

Comment:
If these are known values, we should have them in calculated by default. No need to ask from user.